crypto/keys/ed25519/ed25519.go

```diff
@@ -0,0 +1,234 @@
+package ed25519
+
+import (
+	"crypto/subtle"
+	"fmt"
+	"io"
+
+	"github.com/tendermint/tendermint/crypto"
+	tmed25519 "github.com/tendermint/tendermint/crypto/ed25519"
+	"github.com/tendermint/tendermint/crypto/tmhash"
+	"golang.org/x/crypto/ed25519"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+)
+
+//-------------------------------------
+
+var _ cryptotypes.PrivKey = &PrivKey{}
+var _ codec.AminoMarshaler = &PubKey{}
```

Comment:
add `var _ codec.AminoMarshaler = &PrivKey{}` and `var _ cryptotypes.PubKey = &PubKey{}`?